lesson-3/js/main.js

```diff
@@ -1,65 +1,78 @@
 function palindrome(str) {
-    var strLen = str.length; //переменная, равная длине слова//
-    for (var i = 0; i < strLen; i++) {
-      if (str[i] === str[strLen - 1 - i]) {
-        return true; //сравниваем буквы с разных концов слова//
+  str = str.toLowerCase().replace(/[^а-яa-z]/);
```

Comment:
don't change arguments. Create a new variable in function,equate to an argument and work further with it